modules/installation-azure-config-yaml.adoc

```diff
@@ -41,9 +41,13 @@ controlPlane: <2>
   name: master
   platform:
     azure:
+      encryptionAtHost: true
       osDisk:
         diskSizeGB: 1024 <5>
         diskType: Premium_LRS
+        diskEncryptionSet:
```

Comment:
There is an optional subscriptionId field that can be used here for a disk encryption set in another subscription.